src/20_Components/amp-facebook-like.html

```diff
@@ -11,20 +11,23 @@
 <title>amp-facebook</title>
 <script async src="https://cdn.ampproject.org/v0.js"></script>
 <!-- ## Setup -->
-<!-- Import the amp-facebook-like component to add a like button-->
+<!-- Import the `amp-facebook-like` component to add a like button-->
 <script async custom-element="amp-facebook-like" src="https://cdn.ampproject.org/v0/amp-facebook-like-0.1.js"></script>
 <link rel="canonical" href="<%host%>/components/amp-facebook-like/">
 <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
 <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
 <style amp-custom>
+:root {
+  --space-2: 1rem;   /* 16px */
+}
 amp-facebook-like {
```

Comment:
Can you scope it to this sample

Reply:
Actually, I think we can just remove this.

Reply:
done